File: swimming-contest-software/Model/src/main/java/model/domain/dtos/RaceDetailsDTO.java
package model.domain.dtos;

import model.domain.enums.SwimmingDistances;
import model.domain.enums.SwimmingStyles;

import java.io.Serializable;

public class RaceDetailsDTO implements Serializable {
    SwimmingDistances swimmingDistance;
    SwimmingStyles swimmingStyle;

    public RaceDetailsDTO(SwimmingDistances swimmingDistance, SwimmingStyles swimmingStyle) {
        this.swimmingDistance = swimmingDistance;
        this.swimmingStyle = swimmingStyle;
    }

    public SwimmingDistances getSwimmingDistance() {
        return swimmingDistance;
    }

    public SwimmingStyles getSwimmingStyle() {
        return swimmingStyle;
    }

    @Override
    public String toString() {
        return swimmingDistance + ", " + swimmingStyle + "; ";
    }
}

File: Model/Domain/DTOs/RaceDetailsDTO.cs
using System;
using Model.Domain.Enums;

namespace Model.Domain.DTOs;

[Serializable]
public class RaceDetailsDTO
{
    public RaceDetailsDTO(SwimmingDistance swimmingDistance, SwimmingStyle swimmingStyle)
    {
        SwimmingDistance = swimmingDistance;
        SwimmingStyle = swimmingStyle;
    }

    public SwimmingDistance SwimmingDistance { get; }
    public SwimmingStyle SwimmingStyle { get; }

    public override string ToString()
    {
        return SwimmingDistance + ", " + SwimmingStyle + "; ";
    }
}